shared/helpers/view.js

```diff
@@ -68,7 +68,13 @@ function getClientPlaceholder(viewName, viewOptions) {
 
   // create a list of data attributes
   var attrString = _.inject(viewOptions, function(memo, value, key) {
-    if (_.isArray(value) || _.isObject(value)) { value = JSON.stringify(value); }
+    if (_.isArray(value) || _.isObject(value)) {
+      if (typeof value.toJSON === 'function') {
+        value = value.toJSON();
```

Comment:
bah! i just realized this.. `'[object Object]'` in the template. *sigh* that's why it worked.